public/app/plugins/datasource/grafana-azure-monitor-datasource/components/SubscriptionField.tsx

```diff
@@ -56,9 +56,12 @@ const SubscriptionField: React.FC<SubscriptionFieldProps> = ({
         return;
       }
 
-      query.subscriptions = change.map((c) => c.value ?? '');
+      let newQuery: AzureMonitorQuery = {
```

Comment:
nit: you can either set this to a `const` or simply doing the replacement in `onQueryChange`:

```
      onQueryChange({
        ...query,
        subscriptions: change.map((c) => c.value ?? ''),
      });
```